scripts/release_scripts/get_repo_specific_changes.py

```diff
@@ -0,0 +1,147 @@
+    """Checks if the versions for the exploration or collection schemas have
+    changed.
+
+    Args:
+        current_release: str. The current release tag to diff against.
+
+    Returns:
+        list(str). List of variable names that changed.
+    """
+    feconf_changed_version = []
+    git_show_cmd = (GIT_CMD_SHOW_FORMAT_STRING % current_release)
+    old_feconf = common.run_cmd(git_show_cmd.split(' '))
+    with python_utils.open_file(FECONF_FILEPATH, 'r') as feconf_file:
+        new_feconf = feconf_file.read()
+    for variable in FECONF_VAR_NAMES:
+        old_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, old_feconf)[0]
+        new_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, new_feconf)[0]
+        if old_version != new_version:
+            feconf_changed_version.append(variable)
+    return feconf_changed_version
+
+
+def _git_diff_names_only(left, right='HEAD'):
+    """Get names of changed files from git.
+
+    Args:
+        left: str. Lefthand timepoint.
+        right: str. Rightand timepoint.
+
+    Returns:
+        list(str). List of files that are different between the two points.
+    """
+    diff_cmd = (GIT_CMD_DIFF_NAMES_ONLY_FORMAT_STRING % (left, right))
+    return common.run_cmd(diff_cmd.split(' ')).splitlines()
+
+
+def check_setup_scripts(base_release_tag, changed_only=True):
+    """Check if setup scripts have changed.
+
+    Args:
+        base_release_tag: str. The current release tag to diff against.
+        changed_only: bool. If set to False will return all tested files
+            instead of just the changed ones.
+
+    Returns:
+        dict. Dict consisting of script or boolean indicating whether or not it
+        has changed (filtered by default to those that are modified).
+    """
+    setup_scripts = ['scripts/%s' % item for item in
+                     ['setup.py', 'setup_gae.py', 'install_third_party_libs.py',
+                      'install_third_party.py']]
+    changed_files = _git_diff_names_only(base_release_tag)
+    changes_dict = {script: script in changed_files
+                    for script in setup_scripts}
+    if changed_only:
+        return {name: status for name, status
+                in changes_dict.items() if status}
+    else:
+        return changes_dict
+
+
+def check_storage_models(current_release):
+    """Check if files in core/storage have changed and returns them.
+
+    Args:
+        current_release: str. The current release version.
+
+    Returns:
+        list(str). The changed files (if any).
+    """
+    diff_list = _git_diff_names_only(current_release)
+    return [item for item in diff_list if item.startswith('core/storage')]
+
+
+def get_changes(current_release):
+    """Collects necessary info and dumps it to disk.
+
+    Args:
+        current_release: str. The current release version.
+
+    Returns:
+        list(str). The list of repo specific changes to write to release
```

Comment:
Please clarify what "changes" means here. Typically a change is a Change domain object, so if it's a string you might want to use different / more specific terminology.

Reply:
Done.